### ISSUE TITLE
Rework landing page

### ISSUE THREAD

**Issue by jaimergp** [conda/conda-docs#747]
I feel the landing page of a project with the numbers of `conda` should be more attractive to newcomers. The current landing page is a chunk of text with questionable navigation. The landing page of the `scipy` [docs](https://scipy.org/) could serve as inspiration.

**Comment by kenodegard** [comment]
Absolutely concur, other pages for inspiration:

- [Poetry](https://python-poetry.org/) & [Poetry Docs](https://python-poetry.org/docs)
- [Anvil](https://anvil.works/) & [Anvil Docs](https://anvil.works/docs/overview)
- [Bundler](https://bundler.io/) & [Bundler Docs](https://bundler.io/docs.html)
- [Stripe Docs](https://stripe.com/docs) & [Stripe API Docs](https://stripe.com/docs/api)
- [Dropbox Docs](https://www.dropbox.com/developers/documentation)
- [Twitter Docs](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/overview)

In all of these examples what seems to be the takeaway is that the top landing page should not be a ReadTheDocs site but something else.

**Comment by travishathaway** [comment]
Just so people know, I have also created a similar ticket for the https://github.com/conda/conda repo to update its landing page. A lot of the suggestions  for improvement I've outlined there also apply here. https://github.com/conda/conda/issues/11482

**Comment by kathatherine** [comment]
This is likely going to be replaced with a doc landing page on conda.org (still in early development). That work should be done first before we determine if this is still needed.

**Comment by jaimergp** [comment]
Not stale per se but maybe already tracked somewhere else, so up to you folks!
